float64 gaze_left
float64 gaze_toward
float64 gaze_right
